### ISSUE TITLE
PERMANENT RAMPAGE

### ISSUE THREAD

**Issue by SaundoUteki23** [Iskallia/Vault-public#681]
**Describe the bug**
PERMENANT DMG BOOST WITH RAMPAGE SHAMAN

**To Reproduce**
1. STARTED A BOSS FIGHT (BLUE BLAZE)
2. USED RAMPAGE SHAMAN
3. DIED WITH THE SKILL ON TO A CREEPER EXPLOSION
4. NOW I HAVE THE BUFF ON THE OVERWORLD WITH THE DMG INDICATOR ON THE UI

**Expected behavior**
FOR THE BUFFS AND DEBUFFS TO BE CLEARED WHEN YOU LEAVE THE VAULT

**Info (please complete the following information):**
- World Type: [CAKE SERVER EU10]
- The_Vault Version [1.12.1]
- I ALSO APPLIED THE FOLLOWING JAR FILLES PROVIDED BY DOUWSKY TO FIX MY CONNECTION ISSUES.
Xaeros_Minimap_21.14.1_Forge_1.16.5.jar
XaerosWorldMap_1.16.0_Forge_1.16.5.jar

**Screenshots or Crash Report**
![RAMPAGE BUG](https://user-images.githubusercontent.com/97807974/149636971-e4511452-1871-4397-8e62-9e53d9517cff.PNG)

**Comment by bmemike** [comment]
As a point of curiosity, did you die before defeating the boss or after defeating the boss but before you were teleported  to the overworld?

**Comment by clammish** [comment]
Duplicate of #516.

**Comment by MikeS-MS** [comment]
As an extension to this:

You keep the rampage regardless of the way you die. 
You activate it, then proceed to die either by killing yourself, getting killed by a mob, by a player, or by a boss, it doesn't matter. And when you respawn you suddenly have it permanently, but the status effect is gone. 
The bug happens if you respawn while the ability is still active, if you wait on the respawn screen and respawn after the duration is over it's cleaned up properly.
